pages/docs/sql-schema-declaration.mdx

```diff
@@ -1,44 +1,67 @@
-import { Tab, Tabs } from 'nextra-theme-docs';
+import { Tab, Tabs, FileTree, Callout, Cards, Card } from 'nextra-theme-docs';
 
 # SQL schema declaration
 
-You declare SQL schema in TypeScript in either one `schema.ts` file 
-or you can group them logically in multiple logically grouped files, whichever you prefer, all the freedom!
-
-```plaintext
-📦 <project root>
- └ 📂 src
-    └ 📂 db
-       └ 📜 schema.ts
-```
-
-```plaintext
-📦 <project root>
- └ 📂 src
-    └ 📂 db
-       └ 📂 schema
-          ├ 📜 users.ts
-          ├ 📜 countries.ts
-          ├ 📜 cities.ts
-          ├ 📜 products.ts
-          ├ 📜 clients.ts
-          ├ 📜 enums.ts
-          └ 📜 etc.ts
-```
-```plaintext
-📦 <project root>
- └ 📂 src
-    ├ 📂 get-user
-    │  ├ 📜 user.ts
-    │  └ 📜 handler.ts
-    ├ 📂 get-city
-    │  ├ 📜 city.ts
-    │  └ 📜 handler.ts    
-    ├  ...
-```
-
-You can declare tables, indexes and constraints, foreign keys and enums.  
-Please pay attention to `export` keyword, they are mandatory if you'll be using [drizzle-kit SQL migrations generator](/docs/migrations)
+You can declare your SQL schema directly in TypeScript either in a single `schema.ts` file,
+or you can spread them around — whichever you prefer, all the freedom!
+
+
+<Tabs items={['1 File', 'Separate Files', 'Separate Folders']}>
```

Comment:
We've discussed this internally and we don't wanna introduce complexity here by having `FileTree`, md text works completely fine, let's stick to it

I'm not sure about tabs though, having them makes things compact and my only concern is we lose explicit "you see everything on the screen" straight away

Reply:
Let's remove FileTree and see how it looks with tabs
I think side by side 3 elements could be the best solution here